src/helix-ui/util/position/offsetFunctions.js

```diff
@@ -1,143 +1,143 @@
-export function getCenter (off, ref) {
+function getCenter (off, ref) {
```

Comment:
is there a reason we're taking out the `export` @mcortez1497 ? is it unnecessary?

Reply:
Yea, as far as I understand, it's unneeded since we're doing `export default` on the function map at the bottom